tensorboard/webapp/runs/views/runs_table/regex_edit_dialog_component.scss

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+@import 'tensorboard/webapp/theme/tb_theme';
+
+.example-details {
+  button {
+    background-color: transparent;
+    padding: 0;
+    border: none;
+    cursor: pointer;
+    text-decoration: underline;
+    color: map-get($tb-foreground, link);
+    &:visited {
+      color: map-get($tb-foreground, link-visited);
+    }
```

Comment:
You should use
`@include tb-theme-foreground-prop(link, color);` here and make changes for visited.